aws-lambda/node/serverless-framework/serverless.yml

```diff
@@ -0,0 +1,25 @@
+service: sentry-node
+
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  lambdaHashingVersion: 20201221
+  environment:
+    SENTRY_TRACES_SAMPLE_RATE: '1.0'
+    SENTRY_DSN: ${env:SENTRY_DSN}
+    NODE_OPTIONS: '-r @sentry/serverless/dist/awslambda-auto'
+
+plugins:
+  - serverless-latest-layer-version
+
+custom:
+  layers:
+    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryNodeServerlessSDK:latest
```

Comment:
Oops, note to self: Need to change this back to remove the `latest` stuff - this was some testing for https://github.com/getsentry/sentry-javascript/issues/3834 that slipped in here when doing the folder moves 😬

Reply:
Fixed in https://github.com/getsentry/examples/pull/101/commits/cbc427cd781e65b7eb436b9bd54ddf49f6b9703d